nncf/sparsity/schedulers.py

```diff
@@ -117,14 +115,13 @@ def load_state_dict(self, state_dict):
 
     @property
     def current_sparsity_level(self):
```

Comment:
If `sparsity_target_epoch` is 0, sparsity should be equal to `target_sparsity` (for exponential too)

Reply:
Done.